e2e/test/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -215,6 +215,52 @@ describe("scenarios > visualizations > pivot tables", () => {
     cy.findByText("294").should("not.exist"); // the other one is still hidden
   });
 
+  it("should show standalone values when collapsed to the sub-level grouping (metabase#25250)", () => {
+    const questionDetails = {
+      name: "25250",
+      dataset_query: {
+        type: "query",
+        query: {
+          "source-table": ORDERS_ID,
+          filter: ["<", ["field", ORDERS.CREATED_AT, null], "2016-06-01"],
+          aggregation: [["count"]],
+          breakout: [
+            ["field", ORDERS.CREATED_AT, { "temporal-unit": "month" }],
+            ["field", ORDERS.USER_ID, null],
+            ["field", ORDERS.PRODUCT_ID, null],
+          ],
+        },
+        database: SAMPLE_DB_ID,
+      },
+      display: "pivot",
+      visualization_settings: {
+        "pivot_table.column_split": {
+          rows: [
+            ["field", ORDERS.CREATED_AT, { "temporal-unit": "month" }],
+            ["field", ORDERS.USER_ID, null],
+            ["field", ORDERS.PRODUCT_ID, null],
+          ],
+          columns: [],
+          values: [["aggregation", 0]],
+        },
+        "pivot_table.collapsed_rows": {
+          value: [],
+          rows: [
+            ["field", ORDERS.CREATED_AT, { "temporal-unit": "month" }],
+            ["field", ORDERS.USER_ID, null],
+            ["field", ORDERS.PRODUCT_ID, null],
+          ],
+        },
+      },
+    };
+
+    visitQuestionAdhoc(questionDetails);
+    cy.findByText("1162").should("be.visible");
+    // Collapse "User ID" column
+    cy.findByText("User ID").parent().find(".Icon-dash").click();
+    cy.findByText("1162").should("be.visible");
```

Comment:
Should we also assert the expanded state behaves as expected?

Reply:
Good call! I'll add that in

Reply:
I added some more checks after expanding, but the behavior is also covered in the test before this one. So We should be good